blockstore/splitstore/markset.go

```diff
@@ -28,8 +28,14 @@ type MarkSetEnv interface {
 
 func OpenMarkSetEnv(path string, mtype string) (MarkSetEnv, error) {
 	switch mtype {
-	case "", "bloom":
-		return NewBloomMarkSetEnv()
+	case "bloom":
+		return NewBloomMarkSetEnv(false)
+	case "bloomts":
+		return NewBloomMarkSetEnv(true)
+	case "map":
+		return NewMapMarkSetEnv(false)
+	case "mapts":
+		return NewMapMarkSetEnv(true)
```

Comment:
Can we document that `ts` stands for "thread-safe"? I was trying to find the association with Tipsets all this time!

Reply:
haha, ok will do!

Reply:
done.